src/SimpleExcelReader.php

```diff
@@ -24,16 +25,20 @@ class SimpleExcelReader
 
     private bool $useLimit = false;
 
-    public static function create(string $file)
+    public static function create(string $file, ?string $type = null)
     {
-        return new static($file);
+        return new static($file, $type);
     }
 
-    public function __construct(string $path)
+    public function __construct(string $path, ?string $type = null)
```

Comment:
Instead of using `null` as a default value, use an empty string. That will make the code feel a bit simpler.

Reply:
Makes sense :)